packages/desktop-client/src/components/budget/report/ReportComponents.tsx

```diff
@@ -88,13 +89,29 @@ export function IncomeHeaderMonth() {
 }
 
 type GroupMonthProps = {
+  month: string;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏naming could be improved to be more explicit
```suggestion
  selectedMonth: string;
```

Reply:
Note that this is not the "selected" month. It is the current month being rendered.

This comes from using `RenderMonths`.  If I changed it here, we would need to change it everywhere that uses that component (and there are a lot).

Reply:
That makes sense. Thanks for explaining!